Automattic-Tracks-iOS/TracksService.h

```diff
@@ -12,6 +12,8 @@ extern NSString *const TrackServiceDidSendQueuedEventsNotification;
 @property (nonatomic, strong) TracksServiceRemote *remote;
 
 @property (nonatomic, copy) NSString *eventNamePrefix;
+@property (nonatomic, copy) NSString *anonymousUserType;
+@property (nonatomic, copy) NSString *authenticatedUserType;
```

Comment:
I think these property names aren't clear enough as to what they are. Maybe `anonymousUserTypeKey` and `authenticatedUserTypeKey`?